library/core/src/slice/mod.rs

```diff
@@ -2764,89 +2768,6 @@ impl<T> [T] {
         None
     }
 
-    /// Returns a subslice with the optional prefix removed.
-    ///
-    /// If the slice starts with `prefix`, returns the subslice after the prefix.  If `prefix`
-    /// is empty or the slice does not start with `prefix`, simply returns the original slice.
-    /// If `prefix` is equal to the original slice, returns an empty slice.
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// #![feature(trim_prefix_suffix)]
```

Comment:
Looks like a merge having gone wrong? I don't think this piece should be removed.